invenio_exporter/views.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
understanding check: the exporter is an API only module or will it provide REST endpoints/views? If the former, this file is not needed.